### DISCUSSION TITLE
Community-led paper for The Turing Way

### DISCUSSION THREAD

**Discussion by malvikasharan** [the-turing-way/the-turing-way#2735] [Announcements]
### Summary

* There is still a lack of visibility of The Turing Way in the wider research community -- community-members have expressed interest in co-authoring a paper 
* This paper will be different from a detailed Turing Way paper which will draw from existing reports and proposals, but more generally speak about '**community-led efforts** for building good practice on data science'
* **This issue will discuss what this paper will include** - again the focus should not be to discuss The Turing Way guides as they are already published in the book, but focus on why it is an important resource for the wider research community

### What needs to be done?

- [ ] comment below what you hope this paper can include
- [ ] comment what you want to talk about
- [ ] comment where we can submit this

### Who can help?

* In the initial drafting stage, we would invite those who have previously engaged with The Turing Way closely and would like to share their experience: such as worked in a team within The Turing Way, participated/mentored in a Book Dash, participate/lead a working group, designed a talk/workshop related to The Turing Way, used The Turing Way for their research/teach their team etc.
* In the later stage, we will open the draft up for the wider community for their feedback and reviews

---

**Comment by AoifeHughes** [comment]
Have you thought about where you would want to submit? 

One possibility:
- https://jose.theoj.org

**Comment by TessellateDataScience** [comment]
Why not create a impact-driven digital communication thing instead? Text is such a inefficient way to convince people (i.e. pitching them via a lengthy & dense formal & stuffy document). Off the top of my head I was thinking of building awareness using a TED-inspired talk, but with more ranting, to be posted on Instagram!

The issue appears credit along with visibility?
